src/List.jsx

```diff
@@ -50,12 +50,12 @@ export default class List extends React.Component<ListProps, ListState> {
     return values.findIndex(a => this.getId(a) === this.getId(selectedItem));
   };
 
-  getId = (item: OutputItem | string): string => {
+  getId = (item: textToReplaceType | string): string => {
     if (typeof item === 'string' || typeof item.key === 'undefined') {
       return this.props.getTextToReplace(item).text;
     }
 
-    return item.key;
+    return String(item.key);
```

Comment:
I had to cast to a sting to make flow happy

Reply:
The flow was unsatisfied with our check for `typeof item.key === 'undefined'` so I've fixed this more elegantly I hope (with implicit conversion). 👍